minio/api.py

```diff
@@ -45,8 +45,7 @@
 
 # Internal imports
 from . import __title__, __version__
-from .compat import range  # pylint: disable=redefined-builtin
```

Comment:
If you remove this, `range` usage in this module does not use `xrange` in python2.